File: golang/highest_lowest.go
package kata

import (
		"strconv"
		"strings"
)

func HighAndLow(in string) string {
	slice := strings.Fields(in)
	var hal = []int{}

	for _, i := range slice {
		num, err := strconv.Atoi(i)
		if err != nil {
			panic(err)
		}
		hal = append(hal, num)
	}

	max, min := hal[0], hal[0]

	for _, a := range hal {
		if a > max {
			max = a
		}
		if a < min {
			min = a
		}
	}

	high, low := strconv.Itoa(max), strconv.Itoa(min)

	return high + " " + low
}

File: python/highest_lowest.py
def high_and_low(numbers):
	numbers = numbers.split(" ")
	intnumbers = []
	for number in numbers:
		intnumber = int(number)
		intnumbers.append(intnumber)
	x = max(intnumbers)
	y = min(intnumbers)
	return (str(x) + " " + str(y))